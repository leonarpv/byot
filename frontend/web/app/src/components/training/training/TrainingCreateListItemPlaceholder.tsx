import * as React from 'react';
import {makeStyles, Theme} from '@material-ui/core';
import {WithStyles} from '@byot-frontend/web-common/src/types/WithStyles';
import {AddRounded} from '@material-ui/icons';

interface Props extends WithStyles<typeof styles> {}

const styles = (theme: Theme) => ({
  root: {
    width: '100%',
    height: '100%',
    display: 'flex',
    alignItems: 'center',
    justifyContent: 'center',
  },
});
const useStyles = makeStyles(styles);

export const TrainingCreateListItemPlaceholder: React.FC<Props> = (props: Props) => {
  const styles = useStyles(props);
  return (
    <div className={styles.root}>
      <AddRounded fontSize="large" color="primary" />
    </div>
  );
};
