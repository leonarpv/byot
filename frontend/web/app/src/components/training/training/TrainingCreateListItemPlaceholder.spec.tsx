import * as React from 'react';
import {render} from '@testing-library/react';
import {TrainingCreateListItemPlaceholder} from './TrainingCreateListItemPlaceholder';

describe('<TrainingCreateListItemPlaceholder/>', () => {
  it('should render', () => {
    const {container} = render(<TrainingCreateListItemPlaceholder />);
    expect(container).toMatchSnapshot();
  });
});
