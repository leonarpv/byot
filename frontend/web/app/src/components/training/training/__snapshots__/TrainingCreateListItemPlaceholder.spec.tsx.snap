// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TrainingCreateListItemPlaceholder/> should render 1`] = `
<div>
  <div
    class="makeStyles-root-1"
  >
    <svg
      aria-hidden="true"
      class="MuiSvgIcon-root MuiSvgIcon-colorPrimary MuiSvgIcon-fontSizeLarge"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M18 13h-5v5c0 .55-.45 1-1 1s-1-.45-1-1v-5H6c-.55 0-1-.45-1-1s.45-1 1-1h5V6c0-.55.45-1 1-1s1 .45 1 1v5h5c.55 0 1 .45 1 1s-.45 1-1 1z"
      />
    </svg>
  </div>
</div>
`;
